src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -1213,6 +1213,19 @@ void WindowImplX11::grabFocus()
 }
 
 
+////////////////////////////////////////////////////////////
+void WindowImplX11::setIMEPreeditPosition(const Vector2i& position)
+{
+    XPoint xpos;
+    xpos.x = static_cast<short>(position.x);
+    xpos.y = static_cast<short>(position.y);
+    XVaNestedList preeditAttr;
+    preeditAttr = XVaCreateNestedList(0, XNSpotLocation, &xpos, NULL);
+    XSetICValues(m_inputContext, XNPreeditAttributes, preeditAttr, NULL);
+    XFree(preeditAttr);
```

Comment:
```suggestion
    const XPoint xpos{static_cast<short>(position.x), static_cast<short>(position.y)};
    const XVaNestedList preeditAttr = XVaCreateNestedList(0, XNSpotLocation, &xpos, nullptr);
    XSetICValues(m_inputContext, XNPreeditAttributes, preeditAttr, nullptr);
    XFree(preeditAttr);
```

`const` can be omitted if the other functions do not accept it.

Reply:
`XSetICValues` takes a handle and a variable length argument list, and `XFree` takes a `void*` which is also the type for most X11 handles. Both of them make copies of data, so `const` should be fine, they're good old C functions anyway.